rtcpeerconnection.go

```diff
@@ -1539,8 +1523,8 @@ func (pc *RTCPeerConnection) newRTCTrack(payloadType uint8, ssrc uint32, id, lab
 	}
 
 	trackInput := make(chan media.RTCSample, 15) // Is the buffering needed?
-	rawPackets := make(chan *rtp.Packet)
-	rtcpPackets := make(chan rtcp.Packet)
+	rawPackets := make(chan *rtp.Packet, 15)     // Is the buffering needed?
```

Comment:
I am just making this consistent with the others ones.

I noticed on the SFU example a poorer experience, this needs to be visited fully.